File: Source/AlphaTab/Model/BendStyle.cs
namespace AlphaTab.Model
{
    /// <summary>
    /// Lists the different bend styles
    /// </summary>
    public enum BendStyle
    {
        /// <summary>
        /// The bends are as described by the bend points 
        /// </summary>
        Default,
        /// <summary>
        /// The bends are gradual over the beat duration. 
        /// </summary>
        Gradual,
        /// <summary>
        /// The bends are done fast before the next note. 
        /// </summary>
        Fast
    }
}

File: src/model/BendStyle.ts
/**
 * Lists the different bend styles
 */
export enum BendStyle {
    /**
     * The bends are as described by the bend points
     */
    Default = 0,
    /**
     * The bends are gradual over the beat duration.
     */
    Gradual = 1,
    /**
     * The bends are done fast before the next note.
     */
    Fast = 2
}
